app/request_handler.py

```diff
@@ -49,7 +49,7 @@ def writeYelpRecords(yelpVenues):
 def writeVenueProviderRecord(yelpID, details):
     try:
         venue = representation.updateRecord(yelpID, **details)
-        for provider, data in venue["providers"].items():
+        for provider, data in list(venue["providers"].items()):
```

Comment:
It's not necessary – the tool made these changes because it can't tell the difference and, to be quick about it, I left them.

From one of my commit messages
> In the case of `dict.items/keys/values`, afaict these return iterables and so
wrapping with list is not necessary when iterating over them but is necessary
in cases like:

```python
keys = a_dict.keys()
if 'u' in keys:
  print('i found u!')
```